components/InviteAdminModal/InviteAdminModal.tsx

```diff
@@ -0,0 +1,283 @@
+      const templateParams = { email: inv.email, name: inv.name };
+      sendEmail(templateParams);
+    });
+    reset();
+    setCount(1);
+    setInvites([0]);
+    setSent(true);
+  };
+
+  const addRow = () => {
+    setCount(count + 1);
+    setInvites([...invites, count]);
+  };
+
+  const renderForm = () => {
+    return (
+      <div>
+        <IconButton
+          className={styles["close"]}
+          variant="contained"
+          onClick={() => handleClose()}
+        >
+          <Icon type={"ex"} />
+        </IconButton>
+        <h2 className={styles["title"]}>
+          <Icon className={styles["invite-icon"]} type={"invite"}></Icon>
+          INVITE ADMINS
+        </h2>
+        <form id="inviteForm" onSubmit={handleSubmit(onSubmit)}>
+          <div className={styles["box"]}>
+            <div className={styles["form"]}>
+              <div className={styles["row"]}>
+                <div className={styles["col"]}>
+                  <p className={styles["label"]}> Full Name *</p>
+                </div>
+                <div className={styles["col"]}>
+                  <p className={styles["label"]}>Email *</p>
+                </div>
+              </div>
+
+              {invites.map((i) => (
+                <div key={i} className={styles["row"]}>
+                  <div className={styles["col"]}>
+                    <TextField
+                      inputProps={{ style: { fontSize: 14 } }}
+                      name={"invite." + i + ".name"}
+                      fullWidth
+                      autoFocus
+                      margin="dense"
+                      placeholder="Firstname Lastname"
+                      type="text"
+                      variant="standard"
+                      className={styles["text"]}
+                      {...register("invite." + i + ".name", {
+                        required: true,
+                        maxLength: 5,
+                      })}
+                    />
+                  </div>
+                  <div className={styles["col"]}>
+                    <TextField
+                      name={"invite." + i + ".email"}
+                      inputProps={{ style: { fontSize: 14 } }}
+                      fullWidth
+                      autoFocus
+                      margin="dense"
+                      placeholder="thisisanemail@email.com"
+                      type="text"
+                      variant="standard"
+                      className={styles["text"]}
+                      {...register("invite." + i + ".email")}
+                    />
+                  </div>
+
+                  {i != 0 && (
+                    <IconButton
+                      variant="contained"
+                      onClick={() => removeRow(i)}
+                    >
+                      <Icon type={"inviteTrash"} />
+                    </IconButton>
+                  )}
+                  {i == 0 && <div className={styles["placeholder"]}></div>}
```

Comment:
I think this is a slightly messy implementation to get the spacing aligned (using a placeholder with a fixed amount of pixels). Try playing around with the margins/padding or adjust how your using flexbox